data/campaigns/Winds_of_Fate/scenarios/11_Crosswind.cfg

```diff
@@ -69,7 +69,7 @@
         side=2
         controller=ai
         recruit=Drake Arbiter, Drake Clasher, Drake Thrasher, Drake Fighter, Drake Warrior, Saurian Augur, Saurian Skirmisher, Saurian Ambusher, Saurian Oracle, Saurian Soothsayer
-        gold=0
+        gold=480
```

Comment:
This seems like a pretty big change?

Reply:
It plays only a smaller role late in the scenario due to the:

* size of spawned enemy waves
* enemy leader needing to spend 6 turns getting to her keep before recruiting begins
* size of map further delaying arrival of enemy recruits
* heavy enemy side upkeep costs
* player getting double `village_gold` and `village_support`
* heavy clustering of villages very near to player keep
* value of free ally units

When I play tested it recently I found the last of the three waves of enemies was too weak to create a real challenge. Giving the enemy side 480 gold (same gold player gets on default difficulty) made the last wave just strong enough during a follow up play test.

Maybe 600 or 720 gold would be better but since the stable release is now upon us I wanted to be a bit conservative.

Reply:
Alright, I went ahead and lowered this gold value for all but the hardest difficulty level (Nightmare). So now the default difficulty will have just 120 enemy gold.

Looks good to you, @Pentarctagon ?

Reply:
I can't really judge whether it's a good value, it just stood out to me when looking at the changes.

Reply:
It should be good then. Please merge this PR when you get the chance.